testdata/p4_16_samples/spec-ex19.p4

```diff
@@ -53,7 +53,8 @@ parser Tcp_option_parser(packet_in b,
     }
     state parse_tcp_option_sack {
          b.extract(vec.next.sack,
-                   (bit<32>)(b.lookahead<Tcp_option_sack_top>().length));
+                   (bit<32>)(8 * (b.lookahead<Tcp_option_sack_top>().length) -
```

Comment:
If we want even more error checking, we can consider adding the version attached to a comment of this PR: https://github.com/p4lang/p4-spec/pull/279

If you want me to replace this example with that one, let me know and I will do that.

My purpose in this PR so far was simply to make the program use a more correct argument to extract(), as it is a misleading example as written now.